### ISSUE TITLE
Install Documentation For Windows - Which file is the All-InOne download ?

### ISSUE THREAD

**Issue by hepiyellow** [luarocks/luarocks#1531]
[This documentation page](https://github.com/luarocks/luarocks/wiki/Installation-instructions-for-Windows) for Windows installation says:

> we also provide an all-in-one package which includes Lua 5.1 and LuaRocks in source format; get the [latest win32.zip file here](http://luarocks.github.io/luarocks/releases). The instructions below are about this package.

When opening the link, these are the latest version files:

- luarocks-3.9.2.tar.gz 
- luarocks-3.9.2-windows-32.zip (luarocks.exe stand-alone Windows 32-bit binary)
- luarocks-3.9.2-windows-64.zip (luarocks.exe stand-alone Windows 64-bit binary)
- luarocks-3.9.2-linux-x86_64.zip (luarocks stand-alone Linux x86_64 binary)
- luarocks-3.9.2-win32.zip (legacy Windows package, includes Lua 5.1)

Which one is the all-in-one ?

After downloading all of the zip files, 
I found out that only this one `luarocks-3.9.2-win32.zip ` has the INSTALL.BAT,
so I Assumed this is the all-in-one.
But when I ran the `install.bat`,
I got an error: `Could not find Lua`, so maybe it isn't the all-in-one.

**What's going on?**

**Comment by geomorillo** [comment]
no easy way to install lua with luarocks on windows the docs are confusing

**Comment by svermeulen** [comment]
Yeah, when reading docs it feels like many parts are old.  Examples:
* "For Windows, the all-in-one package already includes them and a binary of Lua, as a convenience." - Lua executable is not included.   The only files are "luarocks.exe" and "luarocks-admin.exe"

Going the install.bat route is similarly confusing

**Comment by RedNeath** [comment]
Weird enough, because it did work for me.

If you try to display the help message you will get a list of options. If you haven't specified it, you should add a `/L` at the end of your command.

```text
...

/L             Install LuaRocks' own copy of Lua even if detected,
               this will always be a 5.1 installation.
               (/LUA, /INC, /LIB, /BIN cannot be used with /L)

...
```
